src/app/mat-main-nav/mobile-menu/mobile-menu.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-accordion displayMode="flat" *ngFor="let menuItem of menuItems$">
+    <mat-expansion-panel *ngIf="menuItem.menuItems">
+        <mat-expansion-panel-header collapsedHeight="48px" expandedHeight="48px">
+            {{menuItem.name}}
+        </mat-expansion-panel-header>
+        <mat-nav-list *ngFor="let item of menuItem.menuItems; let i = index; let last = last">
+            <a mat-list-item *ngIf="item.type === 0" routerLink="['{{ item.route }}']">{{item.name}}</a>
+            <a mat-list-item *ngIf="item.type === 1" [href]="hostURI + item.route">{{item.name}}</a>
+            <!-- show divider every 4th element using modulus -->
+            <mat-divider *ngIf="(i === 3) || (i > 0 && (i+1) % 4 === 0 && !last)"></mat-divider>
+        </mat-nav-list>
+    </mat-expansion-panel>
+
+    <mat-expansion-panel *ngIf="!menuItem.menuItems" hideToggle="true" [ngClass]="'disabled'">
+        <mat-expansion-panel-header collapsedHeight="48px" expandedHeight="48px">
```

Comment:
does the 48px value have to live in the html here or can the same effect be achieved by css?

Reply:
possibly, will investigate. just using their api documentation

Reply:
By default this sets style height to 48px when closed and 64px when open. this causes a flicker on expansion. A css ovveride may have to have the important flag to be able to override this.
![Screenshot 2020-03-16 at 08 43 30](https://user-images.githubusercontent.com/13540819/76738376-6bb98e00-6762-11ea-95c4-861263ec4c91.png)

Reply:
ok no worries, this might be a bit tricky to get right and will probs be a bit time consuming so we can tackle at a later time